### DISCUSSION TITLE
TypeError: Differ in file name casing

### DISCUSSION THREAD

**Discussion by jynnie** [vercel/vercel#5672] [Help]
Has anyone dealt with a rename in file that only differs in casing? Project now hits a type error every time. I can build it locally and use Vercel CLI to deploy, but would ideally like the auto-builds to stop failing.

```
Type error: Already included file name '/vercel/workpath0/.../Form.tsx' differs from 
file name '/vercel/workpath0/.../form.tsx' only in casing.
```

**Comment by fabyeah** [comment]
@jynnie I have the same problem 😟 How did you fix it?

**Reply by fabyeah** [reply]
The file name casing wasn't actually updated on Github.
I fixed it now via `git mv -f OldFileNameCase newfilenamecase`. Pushed again and the deploy worked.

**Reply by tomihq** [reply]
ty so much!
